runscripts/echam/echam-blogin-initial-yearly.yaml

```diff
@@ -8,6 +8,7 @@ general:
         nyear: 1                        # Number of years per run
         #model_dir: /home/shkifmsw/esm/models
         model_dir: /home/shkifmsw/esm/models/echam-${echam.version}
+        use_venv: False
```

Comment:
I agree with all the changes. However, I will remove the following line in the `echam-blogin-initial-yearly.yaml` so that the dialog box about the virtual environment shows up and teaches the user on how to use or not the virtual environment.

Reply:
That venv stuff is accidental. I don't know how it came there but when I commited the changes it went there.